ros2service/ros2service/verb/find.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+from ros2srv.api import service_type_completer
+from ros2service.api import get_service_names_and_types
+from ros2service.verb import VerbExtension
+
+
+class FindVerb(VerbExtension):
+    """Output a list of available services of a given type."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument(
+            'service_type',
+            help="Name of the ROS service type to filter for \
```

Comment:
Instead of using `\` please user two literal string - they will be concatenated automatically.

Reply:
Done.